typescript/packages/subsurface-viewer/src/layers/polylines/polylinesLayer.ts

```diff
@@ -54,6 +51,10 @@ export interface PolylinesLayerProps extends ExtendedLayerProps {
      */
     linesWidth: number;
 
+    /** Enable/disable depth testing when rendering layer. Default true.
+     */
+    depthTest: boolean;
```

Comment:
Suggest making this optional, since there is a sensible default value.

Reply:
Fixed. Now the field is optional

Reply:
Sorry, I don't know how to reply on your comment about npm run docker:storybook:test:update below, have I enough permissions?
Yes, I did. It generated three new images which I submitted but the test fails